src/components/atoms/ImageInput/ImageInput.scss

```
@@ -35,32 +36,50 @@
     font-style: italic;
   }
 
-  &__upload-button {
-    padding: $image-input-upload-button-padding;
-    margin: 0;
-    border-radius: $border-radius--xl;
-    font-weight: bold;
-    transition: $image-input-upload-button-transition;
-    background-color: $image-input-upload-button-background;
-    &:hover,
-    &:active,
-    &:focus {
-      transform: translateY(-1px);
-      background-color: $image-input-upload-button-background--focus;
+  &__upload {
+    &-container {
```

Please don't DRY up selectors like `&__upload-button` into selectors like `&__upload {&-container {`. Don't split names. `__` splits Block from Element and `--` splits element from modifier, but `-` is just a part of a name and it makes it harder to track names and easier to confuse it with the former two occasions.

```suggestion
  &__upload-container {
```

---

thanks, done b18441c